package.json

```diff
@@ -64,12 +64,13 @@
     "sharp": "0.30.7",
     "swagger-cli": "4.0.4",
     "swagger-ui-react": "4.12.0",
+    "zod": "3.17.9",
     "zustand": "3.7.1"
   },
   "devDependencies": {
-    "@chakra-ui/cli": "2.1.2",
     "@babel/core": "7.18.9",
     "@babel/parser": "7.18.9",
+    "@chakra-ui/cli": "2.1.2",
```

Comment:
Why do you add this package ?

Reply:
He just moved it, this package exists since last week.